inc/pendingreason.class.php

```diff
@@ -0,0 +1,275 @@
+<?php
+/**
+ * ---------------------------------------------------------------------
+ * GLPI - Gestionnaire Libre de Parc Informatique
+ * Copyright (C) 2015-2021 Teclib' and contributors.
+ *
+ * http://glpi-project.org
+ *
+ * based on GLPI - Gestionnaire Libre de Parc Informatique
+ * Copyright (C) 2003-2014 by the INDEPNET Development Team.
+ *
+ * ---------------------------------------------------------------------
+ *
+ * LICENSE
+ *
+ * This file is part of GLPI.
+ *
+ * GLPI is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * GLPI is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with GLPI. If not, see <http://www.gnu.org/licenses/>.
+ * ---------------------------------------------------------------------
+ */
+
+if (!defined('GLPI_ROOT')) {
+   die("Sorry. You can't access this file directly");
+}
+
+/**
+ * ITILCategory class
+**/
+class PendingReason extends CommonDropdown
+{
+   // From CommonDBTM
+   public $dohistory = true;
+
+   // From CommonDBTM
+   public $can_be_translated = true;
+
+   // Rights managment
+   public static $rightname = 'pendingreason';
+
+   public static function getTypeName($nb = 0) {
+      return _n('Pending reason', 'Pending reasons', $nb);
+   }
+
+   public function getAdditionalFields() {
+      return [
+         [
+            'name'  => 'followup_frequency',
+            'label' => __('Automatic follow-up'),
+            'type'  => '',
+            'list'  => true
+         ],
+         [
+            'name'      => 'itilfollowuptemplates_id',
+            'label'     => __('Followup template used for automatic follow-up'),
```

Comment:
Not sure it is necessary to have such a long label
```suggestion
            'label'     => __('Followup template'),
```